docs/registering-strategies.adoc

```diff
@@ -0,0 +1,80 @@
+== Registering Custom Strategies
```

Comment:
You haven't included `docs/registering-strategies.adoc` in `README.adoc`

Reply:
Makes me wonder - no one noticed something is not right with the docs preview we automated? Shouldn't we be getting comments from `alien-ike` about docs preview be ready?